docs/pages/code/e.php

```diff
@@ -15,9 +15,11 @@
 
 $addition = static fn (float $value1, float $value2): float => $value1 + $value2;
 
-$fact = static fn (float $number): float => (float) Collection::range(1, $number + 1)->foldLeft($multiplication, 1)->current();
+$fact = static fn (float $number): float => (float) Collection::range(1, $number + 1)
+    ->foldLeft($multiplication, 1)
+    ->current();
```

Comment:
[This change](https://github.com/loophp/collection/commit/3f1855395c40675e8f9cf3eac070a2a82c867ab4) was causing a[ failure in CI](https://github.com/loophp/collection/runs/2687478640?check_suite_focus=true) and confusing me massively 😄

Reply:
Yeah sometimes those tools are extremely frustrating!